server/src/main/scala/geotrellis/admin/server/AdminActor.scala

```diff
@@ -170,11 +188,11 @@ trait AdminRoutes extends HttpService with CORSSupport {
       val layerId = LayerId(layer, zoom)
 
       pathPrefix("grid")(tmsGrid(layerId, key)) ~
-      pathPrefix("type")(tmsType(layerId, key)) ~
-      pathPrefix("breaks")(tmsBreaks(layerId, key)) ~
-      pathPrefix("histo")(tmsHisto(layerId, key)) ~
-      pathPrefix("stats")(tmsStats(layerId, key)) ~
-      pathEnd(serveTile(layerId, key))
+        pathPrefix("type")(tmsType(layerId, key)) ~
```

Comment:
Is this indentation intentional? In general, I think it's a good idea, but I find these path specifications most readable when indentation suggests going a step deeper in context - these just seem to be sibling routes